qrenderdoc/Windows/ShaderViewer.ui

```diff
@@ -319,6 +319,29 @@
       </property>
      </widget>
     </item>
+    <item>
+     <widget class="QToolButton" name="bookmark">
+      <property name="toolTip">
+       <string>Run or step backwards in execution</string>
+      </property>
+      <property name="text">
+       <string>Bookmark...</string>
+      </property>
+      <property name="icon">
+       <iconset resource="../Resources/resources.qrc">
+        <normaloff>:/bookmark_blue@x2.png</normaloff>:/bookmark_blue@x2.png</iconset>
```

Comment:
The icon referenced by default should be the normal version, the `@2x` versions will be automatically substituted by Qt if the dpi ratio is high enough.

Reply:
I can't believe I misnamed those...  Time to change my glasses!